src/components/books.jsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable import/extensions */
+import React, { useState } from 'react';
+import Book from './Book.jsx';
+import Form from './Form';
+
+function Books() {
+  const [books, setBooks] = useState([
+    {
+      id: 1,
+      title: 'Vulture is a Patient Bird',
+      author: 'James Hardley Chase',
+    },
+    {
+      id: 2,
+      title: 'A Time to Kill',
+      author: 'John Grisham',
+    },
+    {
+      id: 3,
+      title: 'Good Times',
+      author: 'Kola Kolade',
+    },
+  ]);
+  return (
+    <div>
+      <h3>Books</h3>
+      <>
+        <div className="book-list">
+          <ul className="list">
+            <Book books={books} setBooks={setBooks} />
+          </ul>
+        </div>
```

Comment:
- [Optional] Dear @kkolade. I see you take a bit of a different approach for rendering the books. But wouldn't it make sense if the `books.jsx` component implement the `books.map((book)` method and the `book.jsx` component be responsible for only rendering a single book by taking a `title` and `author` of a single book. I hope you would make the change, don't hesitate to ask for clarification if anything is not clear.